apps/web/pages/export.tsx

```diff
@@ -111,6 +111,9 @@ interface SyncSpreadsheetCardProps {
 
 function SyncSpreadsheetCard(props: SyncSpreadsheetCardProps) {
   const {apiKey, isEmptyResult, selectedTable} = props
+  if (!selectedTable) {
+    return <></>
```

Comment:
@iwasrobbed fyi `return null` is idiomatic

Reply:
Great info, thank you for that!